pywbem/_server.py

```diff
@@ -776,6 +806,14 @@ def get_central_instances(self, profile_path, central_class=None,
               :class:`~pywbem.CIMInstanceName`.
         """  # noqa: E501
         # pylint: enable=line-too-long
+
+        if reference_direction != ('dmtf', 'snia'):
```

Comment:
I don't think this comparison works. What you want, is: 
```
if reference_direction not in ('dmtf', 'snia'):
```

Reply:
That is what I get for testing against the version in the test scripts. Fixed